cloudigrade/config/settings/base.py

```diff
@@ -155,4 +155,4 @@
 
 RABBITMQ_EXCHANGE_NAME = 'cloudigrade_inspectigrade'
 RABBITMQ_QUEUE_NAME = 'machine_images'
-RABBITMQ_URL = 'amqp://guest:guest@localhost:5672/%2F'
+CELERY_BROKER_URL = 'amqp://guest:guest@localhost:5672/%2F'
```

Comment:
Out of curiosity, did you ever encounter a need to set `CELERY_RESULT_BACKEND`? It might not be necessary since we aren't checking for results from calling a task, but when I was experimenting locally, I *think* I had to set `CELERY_RESULT_BACKEND` to the same value as `CELERY_BROKER_URL` to make that work.

Reply:
I have not yet. Since our current design chains tasks and uses message queues as go-betweens, I didn't see a need to track the result directly here. I believe Celery is smart enough to ignore results until you give it a back end.

Reply:
Related to the previous comment about including the latest change from master, this config's value should also be pulled via `env`.